_includes/codeTable.html

```diff
@@ -140,6 +141,14 @@
             <td>{{ administration[0].name[page.lang] }}</td>
 
             <td><a href="#{{ release.name[page.lang] | slugify }}" aria-controls="{{ release.name[page.lang] | slugify }}" class="overlay-lnk" role="button" >{{ release.name[page.lang] }}</a></td>
+            <td>
+              {%- assign cat = release.category -%}
+              {%- for category in site.data.i18n.categories -%}
+                  {%- if category[1].value == cat -%}
```

Comment:
This is more complicated than it should. You can access the category by using directly `site.data.i18n.categories[release.category]` or smth similar.

Reply:
Resolved